### PR TITLE
Fpe Feedback Changes

### PR DESCRIPTION
**Pull request by HariharanIT**
# Related Issue
- Mention your issue description here
- clickup ticket

# Propossed changes/Fix
- Change 1
- Change 2
- ...

# Additional Info 
- Any additional info or context for the reviewer

# Checklist
- [ ] Tested on local/dev branch for all major platforms (Android, IOS, Desktop, Web).
- [ ] No commented out code
- [ ] Is any third party library, service used
- [ ] Tests
- [ ] If this change requires updates outside of the code, like updates in core, react-ui-kit, RTM/RTC configure.

# Dependent PRs 
- Mention dependent PR links.

# Screenshots

Original                |          Updated
:---------------------:  | :-----------------------:
**original screenshot** |  **updated screenshot**

### CODE REVIEW

**Review by nitte93 — CHANGES_REQUESTED** [review]

File: template/src/components/Share.tsx [hunk 1]
```diff
@@ -148,6 +53,25 @@ const Share = () => {
...
+  const getAttendeeLabel = () => (isWeb ? attendeeUrlLabel : attendeeIdLabel);
+
+  const getHostLabel = () => {
+    if (isSeparateHostLink) {
+      if (isWeb) {
+        return hostUrlLabel;
+      } else {
+        return hostIdText;
+      }
+    } else {
```
[the remark this answers is not in the document]
Comment (HariharanIT) [reply template/src/components/Share.tsx:67]: Done

File: template/src/components/Share.tsx [hunk 2]
```diff
@@ -148,6 +53,25 @@ const Share = () => {
...
 
+  const isWeb = $config.FRONTEND_ENDPOINT || platform === 'web';
+
+  const getAttendeeLabel = () => (isWeb ? attendeeUrlLabel : attendeeIdLabel);
+
+  const getHostLabel = () => {
+    if (isSeparateHostLink) {
+      if (isWeb) {
+        return hostUrlLabel;
+      } else {
```
[the remark this answers is not in the document]
Comment (HariharanIT) [reply template/src/components/Share.tsx:64]: Done

File: template/src/pages/video-call/useLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/*
+********************************************
+ Copyright © 2021 Agora Lab, Inc., all rights reserved.
+ AppBuilder and all associated components, source code, APIs, services, and documentation 
+ (the “Materials”) are owned by Agora Lab, Inc. and its licensors. The Materials may not be 
+ accessed, used, modified, or distributed for any purpose without a license from Agora Lab, Inc.  
+ Use without a license or in violation of any license terms and conditions (including use for 
+ any purpose competitive to Agora Lab, Inc.’s business) is strictly prohibited. For more 
+ information visit https://appbuilder.agora.io.
```
[the remark this answers is not in the document]
Comment (HariharanIT) [reply template/src/pages/video-call/useLayout.tsx:9]: done

File: template/src/components/Navbar.tsx [hunk 4]
```diff
@@ -52,14 +54,9 @@ const Navbar = () => {
...
-    setSidePanel,
-    activeLayoutName,
-    setActiveLayoutName,
-    isHost,
-    title,
-  } = useVideoCall((data) => data);
+  const {activeLayoutName} = useLayout((data) => data);
+  const {sidePanel, setSidePanel} = useSidePanel();
+  const {meetingTitle, isHost} = useMeetingInfo();
   const {recordingActive} = useRecording((data) => data);
```
[the remark this answers is not in the document]
Comment (HariharanIT) [reply template/src/components/Navbar.tsx:60]: done

File: template/src/components/Share.tsx [hunk 5]
```diff
@@ -148,6 +53,25 @@ const Share = () => {
     setDim([e.nativeEvent.layout.width, e.nativeEvent.layout.height]);
   };
 
+  const isWeb = $config.FRONTEND_ENDPOINT || platform === 'web';
```
[the remark this answers is not in the document]
Comment (HariharanIT) [reply template/src/components/Share.tsx:56]: we need to find out whether it's a native or web. We also need to FRONTEND_ENDPOINT. That is the reason for having a custom check.

**Pull request merged**